pkg/policy/api/ingress.go

```diff
@@ -65,6 +65,15 @@ type IngressCommonRule struct {
 	// +kubebuilder:validation:Optional
 	FromCIDRSet CIDRRuleSlice `json:"fromCIDRSet,omitempty"`
 
+	// FromCIDRGroupRef is a list of CIDRs which the endpoint subject to the
```

Comment:
Changing this will also update the yaml files above after running `make generate-k8s-api`. I think this sentence is fine for both `Ingress` and `IngressDeny`. :+1: